### PR TITLE
Changing Notification Admin to OrgAdmin

### PR DESCRIPTION
**Pull request by ebenezergraham**
Pull Request contains code for Permitted Feign Client as well. 
Contains additions to API documentations

### CODE REVIEW

**Comment by awasum** [comment]
Please Ignore the out folder with byte code.
Add travis to this repo: this is a separate issue. Check out what Juhan has done and copy paste.

File: .gitignore [hunk 1]
```diff
@@ -2,13 +2,8 @@
...
 build/
 target/
-build/out
-service/out
-component-test/out
-api/out
-service/build
-component-test/build
-api/build
+**/out/
```
Comment (ebenezergraham) [inline .gitignore]: @awasum the out directory has been ignored here. :)

Comment (awasum) [reply .gitignore]: OK, thanks. I missed it.

**Comment by awasum** [comment]
@Izakey Will you be able to test and merge if all is well?

**Comment by awasum** [comment]
Fix the conflicts let merge this. You created another PR with same details as this..which one are we working on now... Moreover your PR seems to contain too many distinct features. Code one feature at a time.

**Comment by ebenezergraham** [comment]
> Fix the conflicts let merge this. You created another PR with same details as this..which one are we working on now... 

I am currently working on PR [#11](https://github.com/apache/fineract-cn-notifications/pull/11)
PR [#11](https://github.com/apache/fineract-cn-notifications/pull/11) contains new comments which implement html templating with thymeleaf. Once this has been merged I will pull and sync with the develop branch

> Moreover your PR seems to contain too many distinct features. Code one feature at a time.

Yes, it's a practice I'm learning to develop. :)

**Comment by awasum** [comment]
Is this ready to be merged?

**Comment by awasum** [comment]
there is a conflict here

**Comment by awasum** [comment]
Fix this conflicts @ebenezergraham , lets merge this and move the project forward. Can you do this tomorrow?

**Comment by ebenezergraham** [comment]
@awasum I have resolved the conflict. We can merge now

**Pull request merged**